packages/firebase_crashlytics/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ version: 0.2.0
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_crashlytics
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.8.0 <3.0.0"
   flutter: ">=1.12.13+hotfix.4 <2.0.0"
```

Comment:
I think this should be `1.17.0` as version `1.12.13` has lower dart SDK than `2.8.0`

Reply:
That's right, thanks!